analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -392,35 +376,21 @@ def behavior(self):
 def get_query(source: ObjectStream) -> ObjectStream:
     """Query for event / column selection: >=4j >=1b, ==1 lep with pT>25 GeV, return relevant columns
     """
-    return source.Where(lambda e:
-        # == 1 lep
-        e.electron_pt.Where(lambda pT: pT > 25).Count() + e.muon_pt.Where(lambda pT: pT > 25).Count()== 1
-        )\
-        .Where(lambda e:\
-            # >= 4 jets
-            e.jet_pt.Where(lambda pT: pT > 25).Count() >= 4
-        )\
-        .Where(lambda e:\
-            # >= 1 jet with pT > 25 GeV and b-tag >= 0.5
-            {"pT": e.jet_pt, "btag": e.jet_btag}.Zip().Where(lambda jet: jet.btag >= 0.5 and jet.pT > 25).Count() >= 1
-        )\
-        .Select(lambda e:\
-            # return columns
-            {
-                "electron_e": e.electron_e,
-                "electron_pt": e.electron_pt,
-                "muon_e": e.muon_e,
-                "muon_pt": e.muon_pt,
-                "jet_e": e.jet_e,
-                "jet_pt": e.jet_pt,
-                "jet_eta": e.jet_eta,
-                "jet_phi": e.jet_phi,
-                "jet_btag": e.jet_btag,
-                "numbermuon": e.numbermuon,
-                "numberelectron": e.numberelectron,
-                "numberjet": e.numberjet,
-            }
-        )
+    return source.Where(lambda e: e.Electron_pt.Where(lambda pt: pt > 25).Count() 
+                        + e.Muon_pt.Where(lambda pt: pt > 25).Count() == 1)\
+                 .Where(lambda e: e.Jet_pt.Where(lambda pt: pt > 25).Count() >= 4)\
+                 .Where(lambda g: {"pt": g.Jet_pt, 
+                                   "btagCSVV2": g.Jet_btagCSVV2}.Zip().Where(lambda jet: 
+                                                                             jet.btagCSVV2 >= 0.5 
+                                                                             and jet.pt > 25).Count() >= 1)\
+                 .Select(lambda f: {"Electron_pt": f.Electron_pt,
```

Comment:
Regarding the names used here for the event: I remember that there was a bug at some point where using `e` for all of them didn't work, but as far as I know that is resolved (and `e` everywhere works fine). Could we please either use `e` everywhere or do something like `e`, `f`, `g`, `h` or so? At the moment it looks like there might be some purpose to `e`, `e`, `g`, `f`, but if there is I don't understand it.

Reply:
I will change to the latter option, because when everything is `e`, I get `ValueError: Found multiple calls to on same line - split the calls across lines or change lambda argument names so they are different.`

Reply:
found the issue: https://github.com/iris-hep/func_adl/issues/115, might be resolved by a newer version (but changing name is also fine)